src/properties.ts

```diff
@@ -332,30 +354,39 @@ export function getProperties(
     };
 
     const baseSonarExclusions =
-      cliProperties[ScannerProperty.SonarExclusions] ??
-      scanOptionsProperties[ScannerProperty.SonarExclusions] ??
-      envProperties[ScannerProperty.SonarExclusions] ??
-      DEFAULT_SONAR_EXCLUSIONS;
+      userProperties[ScannerProperty.SonarExclusions] ?? DEFAULT_SONAR_EXCLUSIONS;
 
     inferredProperties = {
       ...inferredProperties,
       ...getPackageJsonProperties(projectBaseDir, baseSonarExclusions),
     };
   }
 
+  // Generate proxy properties from HTTP[S]_PROXY env variables, if not already set
+  const httpProxyProperties = getHttpProxyEnvProperties(
+    userProperties[ScannerProperty.SonarHostUrl],
+  );
+
   // Merge properties respecting order of precedence
-  const properties = {
+  let properties = {
```

Comment:
am I missing something?
```suggestion
  const properties = {
```

Reply:
nope, thanks fixed it 😁